tests/align.rs

```diff
@@ -23,7 +23,7 @@ fn build_cli() {
         .arg(rfile_name)
         .arg("-o")
         .arg("basic_build_opts")
-        .args(&["-v", "--threads", "2", "-k", "31"])
+        .args(&["-v", "--threads", "2", "-k", "31", "--max-reads", "10000"])
```

Comment:
I think this would be better added as new test in `test/build.rs` which uses reads, and with a smaller number so you can actually test the output changes